src/messages/datastore/conversations.js

```diff
@@ -191,7 +191,7 @@ export default {
       switch (type) {
         case 'group': return rootGetters['groups/get'](targetId)
         case 'place': return rootGetters['places/get'](targetId)
-        case 'pickup': return rootGetters['pickups/get'](targetId)
+        case 'activity': return rootGetters['activities/get'](targetId)
```

Comment:
Don't change yet, conversation type is part of API (backend sends it to us)

Reply:
Fixed!